src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/ApiKeysTest.java

```diff
@@ -212,14 +212,21 @@ public void nellaPaginaApiKeyPosizionareIlCursoreSulloStatoDellOperazione() {
     @And("Nella sezione genera Api Key inserire un gruppo")
     public void nellaSezioneGeneraApiKeyInserireUnGruppo() {
```

Comment:
switch table of notification with cucumber table

Reply:
I didn't understand your request